qiskit/aqua/operators/evolutions/evolved_op.py

```diff
@@ -88,9 +88,20 @@ def tensor(self, other: OperatorBase) -> OperatorBase:
 
         return TensoredOp([self, other])
 
-    def compose(self, other: OperatorBase) -> OperatorBase:
-        other = self._check_zero_for_composition_and_expand(other)
+    def _expand_dim(self, num_qubits: int) -> OperatorBase:
+        from qiskit.aqua.operators import I
+        return self.tensor(I ^ num_qubits)
 
+    def permute(self, permutation: List[int]) -> OperatorBase:
```

Comment:
Hi, the docstrings for ```compose``` and ```permute``` are provided in OperatorBase,
and e.g. MatrixOp inherits the docstring documentation.https://qiskit.org/documentation/stubs/qiskit.aqua.operators.primitive_ops.MatrixOp.compose.html#qiskit.aqua.operators.primitive_ops.MatrixOp.compose

I am not sure if it is necessary to override docstrings for all ```compose``` methods (and for ```permute``` as well), because we were not doing it so far.

But if you are sure that it is the right move, I can do it.